include/xrpl/basics/TaggedCache.ipp

```diff
@@ -330,12 +339,13 @@ TaggedCache<
                 m_cache.map()[p],
                 allStuffToSweep[p],
                 allRemovals,
-                lock));
+                partitionLocks_[p]));
         }
         for (std::thread& worker : workers)
             worker.join();
 
-        m_cache_count -= allRemovals;
+        int removals = allRemovals.load(std::memory_order_relaxed);
```

Comment:
Do we want to make it stricter? The value gets modified in `sweepHelper` and if `allRemovals.load(std::memory_order_relaxed)` gets reordered before `sweepHelper` gets called, it'll be 0

Reply:
Actually, if this is the only place that calls `sweepHelper`, we should make allRemovals an int instead of an atomic_int